src/Routes/Api.ts

```diff
@@ -97,6 +97,11 @@ export function createApiRoutes(
         [validator.body(CreateNodeBalanceValidationSchema), passNodeData, AuthorizeUser],
         nodeBalanceController.storeNodeBalance.bind(nodeBalanceController));
 
+    router.get(
+        "/user/node/balance/:nodeId",
```

Comment:
I am not sure how many endpoint we have like this but usually the id is after the attribute i.e. `/user/node/:nodeId/balance/`. Leave like this as other endpoint have this already but just have that in mind for now :)